kivy/uix/video.py

```diff
@@ -31,21 +42,51 @@ def on_duration_change(instance, value):
 from kivy.core.video import Video as CoreVideo
 from kivy.resources import resource_find
 from kivy.properties import (BooleanProperty, NumericProperty, ObjectProperty,
-                             OptionProperty)
+                             OptionProperty, StringProperty)
+from kivy.logger import Logger
 
 
 class Video(Image):
     '''Video class. See module documentation for more information.
     '''
 
+    source = StringProperty(None)
```

Comment:
I think this should default to `""`, rather than None. This is supposed to be strings so there's no reason to use `None` as signal to mean not set.

Reply:
I copied it over from Image. Thus i left it None for consistency reasons

Reply:
Oh, ok. Then we should add `allownone=True`, I think. Otherwise it's weird that you can initially set it to None, but not afterwards.

Reply:
done